main.go

```diff
@@ -134,10 +109,45 @@ func main() {
 	log.Info("rqlite server stopped")
 }
 
+func closeFile(f *os.File) {
+	if err := f.Close(); err != nil {
+		log.Errorf("Unable to close file: %s", err.Error())
+		os.Exit(1)
+	}
+}
+
+func createFile(path string) *os.File {
```

Comment:
Does it make sense to you? We can always create a new file in the root directory to store these helpers, e.g. `helpers.go`, and keep `main.go` cleaner.